client/my-sites/customer-home/celebrate-launch-modal.jsx

```diff
@@ -0,0 +1,53 @@
+import { ConfettiAnimation, Gridicon } from '@automattic/components';
```

Comment:
I see we're just adding the component file and associate css directly to the customer-home folder. Probably not a requirement, but I was wondering if it be cleaner to put these inside a components folder, or otherwise separate them out of the root of customer home?

Reply:
`components` folder is definitely a good idea, I'm on it